application/config/constants.php

```diff
@@ -99,8 +99,18 @@
 /**
  * Stripe public and private keys for development.
  */
-define('STRIPE_PUBLIC_KEY', 'pk_test_A4wjqVPPn530rgAXv6sHKgSl00opCMVX9A');
-define('STRIPE_PRIVATE_KEY', 'sk_test_OMC00A11yJUakUU4kx6KoGTp0028EYnLBa');
+if (!isset($STRIPE_PUBLIC_KEY)) {
+    define('STRIPE_PUBLIC_KEY', 'pk_test_A4wjqVPPn530rgAXv6sHKgSl00opCMVX9A');
+    define('STRIPE_PRIVATE_KEY', 'sk_test_OMC00A11yJUakUU4kx6KoGTp0028EYnLBa');
+}
```

Comment:
Just cleaning up the old code to be more robust in these lines. 

This is confirmed to work on `dev` environment, and should work on `prod`.

Reply:
Good catch!